src/workspacer.Shared/Workspace/IWorkspace.cs

```diff
@@ -1,147 +1,127 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using System.Collections.Generic;
 
 namespace workspacer
 {
     /// <summary>
     /// IWorkspace provides a common interface for workspace-related operations.
-    /// workspaces logically contain a set of windows, and allow callers to interact with the 
+    /// Workspaces logically contain a set of windows, and allow callers to interact with the 
     /// windows via a set of methods, and all the organization of the windows via layout engines
     /// </summary>
     public interface IWorkspace
     {
         /// <summary>
-        /// name of the workspace
+        /// Name of the workspace
         /// </summary>
         string Name { get; set; }
 
         /// <summary>
-        /// name of the currently active layout
+        /// Name of the currently active layout
         /// </summary>
         string LayoutName { get; }
 
         /// <summary>
-        /// set of windows that are contained within the workspace
+        /// Set of windows that are contained within the workspace
         /// </summary>
         IEnumerable<IWindow> Windows { get; }
 
         /// <summary>
-        /// set of windows that are contained within the workspace and which workspacer can layout.
+        /// Set of windows that are contained within the workspace and which workspacer can layout.
         /// </summary>
         IList<IWindow> ManagedWindows { get; }
 
         /// <summary>
-        /// currently focused window in the workspace, or null if there is no focused window in the workspace
+        /// Currently focused window in the workspace, or null if there is no focused window in the workspace
         /// </summary>
         IWindow FocusedWindow { get; }
 
         /// <summary>
-        /// the last focused window in the workspace
+        /// The last focused window in the workspace
         /// </summary>
         IWindow LastFocusedWindow { get; }
 
         /// <summary>
-        /// whether the workspace is currently indicating (flashing)
+        /// Whether the workspace is currently indicating (flashing)
```

Comment:
Lots of changes in code-docs makes it harder to identify the actual bits of code changed, and how they have been changed. It would be easier to review this draft without such changes.